R/actions.R

```diff
@@ -5,7 +5,10 @@ in_github_actions <- function() {
 # Output logging commands for any lints found
 github_actions_log_lints <- function(lints) {
   for (x in lints) {
-    cat(sprintf("::warning file=%s,line=%s,col=%s::%s\n", x$filename, x$line_number, x$column_number, x$message), sep = "")
+    cat(
```

Comment:
should we nocov this?

Reply:
Would be fine with it. Could also add a test similar to `trim_output()`, shouldn't be too hard because it only `cat()`s, so `testthat` can capture and check its output.

Reply:
For some reason I assumed we wouldn't be able to get this working without actually being on GHA. But we can imitate that by messing with env & options, so I added a test.